refinery/analysis_manager/tasks.py

```diff
@@ -46,64 +48,240 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
         analysis.set_status(Analysis.FAILURE_STATUS, error_msg)
 
 
-@task(base=AnalysisHandlerTask, max_retries=None)
-def run_analysis(analysis_uuid):
-    """Manage analysis execution"""
-    RETRY_INTERVAL = 5  # seconds
-
+def _get_analysis(analysis_uuid):
```

Comment:
I'd remove the logic for updating the task state change or even simply get rid of the function completely because it is used only once.